srcpkgs/docker-cli/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'docker-cli'
+pkgname=docker-cli
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
```

Comment:
This is pretty much all platforms we build Go applications for? Not sure it makes sense to restrict...